app/javascript/components/applications/ApplicationView.jsx

```diff
@@ -162,16 +174,10 @@ class ApplicationView extends React.Component {
                     {profileImage}
                     <div className="w-100 m3 ph4 pt4">
                         <div className="flex items-end">
-                            <h1 className="ma0 f1 mb3">
+                            <h1 className="ma0 f1 icon-link mb3" onClick={this.handleVolunteerClick}>
                                 {this.props.user.first_name} {this.props.user.last_name}
                             </h1>
-                            <br/>
-                            <a className="pa0 ph1 ml3" style={{marginBottom: 21}} target="_blank" href={`http://${this.props.user.linkedin_url}`}>
-                            <i className="fab fa-linkedin f2 icon-link"></i>
-                            </a>
-                            <a className="pa0 ph1 ml3" style={{marginBottom: 23}} target="_blank" href={resumeUrl}>
-                            <i className="fas fa-file-alt f2 icon-link"></i>
-                            </a>
+                            {linkedin} {resume}
                         </div>
                         <div className="flex f5 b lato">
```

Comment:
I think you have `{this.props.user.skills}` listed in the render function, but this should be rendered with the skill pills.